docs/en/connector-v2/sink/Druid.md

```diff
@@ -0,0 +1,101 @@
+# Druid
```

Comment:
Please update doc to match new format, reference: https://github.com/apache/incubator-seatunnel/pull/2625